populus/compilation/__init__.py

```diff
@@ -35,7 +35,7 @@ def compile_project_contracts(project):
     compiler_backend = project.get_compiler_backend()
     compiled_contract_data = compiler_backend.get_compiled_contract_data(
         source_file_paths=all_source_paths,
-        import_remappings=None,
+        import_remappings=project._project_config['compilation'].get('import_remappings',None)
```

Comment:
This should be changed to  `project.config.get('compilation.import_remappings', None)`